app-catalog/src/settings.tsx

```diff
@@ -0,0 +1,96 @@
+import { HoverInfoLabel, NameValueTable } from '@kinvolk/headlamp-plugin/lib/components/common';
+import { Box, Switch, SwitchProps, Typography, useTheme } from '@mui/material';
+import { useState } from 'react';
+
+export function AppCatalogSettings() {
+  const [showVerified, setShowVerified] = useState<boolean>(() =>
+    JSON.parse(localStorage.getItem('showVerified') ?? 'true')
+  );
+
+  function toggleShowVerified() {
+    setShowVerified(!showVerified);
+    localStorage.setItem('showVerified', `${!showVerified}`);
```

Comment:
Please check the change-logo example plugin to see how it's using a provided function for setting/getting settings:
```typescript
export const store = new ConfigStore<pluginConfig>('change-logo');
```

Reply:
Done!